effekt/shared/src/main/scala/effekt/context/Context.scala

```diff
@@ -40,11 +41,14 @@ abstract class Context(val positions: Positions)
     extends NamerOps
     with TyperOps
     with ModuleDB
-    with TransformerOps {
+    with TransformerOps
+    with Timers {
 
   // bring the context itself in scope
   implicit val context: Context = this
 
+  var timersActive: Boolean = false
```

Comment:
Can or should this be part of `Timers`?

In `ErrorReporting` and `messaging` (see call below in `setup`). This is similar. The field is part of the cake-slice and `setup` needs to call the initialization method.